dist/Renderer.js

```diff
@@ -0,0 +1,25 @@
+
+class Renderer {
+    renderMainPage(data) {
+        const source = $('#main-template').html()
+        const template = Handlebars.compile(source)
+        $('.container').empty()
+        const mainHtml = template(data)
+        $('.container').append(mainHtml)
+    }
+
+    renderScore1(player1Score) {
```

Comment:
maybe try making only one render method for score next time. Would be very similar and you do not have to repeat yourself

Reply:
Ok i try to do it